asf_heat_pump_suitability/pipeline/reweight_epc/prepare_target.py

```diff
@@ -1,3 +1,7 @@
+"""
```

Comment:
I updated some documentation here.

The main substantial changes are in 2 functions: `get_dict_target_marginals` and `get_dict_dfs_counts`